components/aboutUs/styles.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
```

Comment:
@Srish-ty, all styles will be in the styles folder
- component folder name starts with a capital letter